internal/constants/constants.go

```diff
@@ -81,6 +81,7 @@ const (
 	KibanaTrustedCAName        = "kibana-trusted-ca-bundle"
 	ElasticsearchFQDN          = "elasticsearch"
 	ElasticsearchName          = "elasticsearch"
+	LokiName                   = "loki"
```

Comment:
Consistent with other names, LokiName should be the name prefix for loki components:
```suggestion
	LokiName                   = "logging-loki"
```

Reply:
@alanconway we are at the mercy of someone deploying loki and have no control of the name.  This name comes from the loki operator and is the name of the container.